test/integration/golden_response/staging/get_triples/stat_var.json

```diff
@@ -8,7 +8,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/d7tbsb1"
+      "provenanceId": "dc/kyzlg31"
```

Comment:
this provenance change does not seem right

Reply:
Yes, not expected... something regressed, I'll take a look.

Reply:
The root cause is cl/371847183 introduced a new import tagged as CURATED_STATVAR.

But the logic that assigns StatVar provenances assumed there was one CURATED_STATVAR entry only (http://google3/datacommons/import/util/flume_helper.cc?l=528-531&rcl=371903667). I'll fix it now and get it into tonight's build.

Should we defer data rollout to tomorrow?

Reply:
defer sg, maybe just release the migration clean up?

Reply:
thanks for catching this -- i'll close the PR and just push the migration cleanup